src/track.c

```diff
@@ -818,10 +824,10 @@ int nav_meas_cmp(const void *a, const void *b)
  * array at least `MIN(n_new, n_old)` long will ensure sufficient space.
  *
  * \param n_new Number of measurements in `m_new`
- * \oaram m_new Array of new navigation measurements
+ * \param m_new Array of new navigation measurements
```

Comment:
Ehhh, try to keep unrelated changes into a separate PR (advice for the future). This PR here is mostly a tooling change, right?